runtime/integration-tests/src/cases/liquidity_pools.rs

```diff
@@ -2949,10 +2942,7 @@ mod foreign_investments {
 								investor: investor.clone().into(),
 								currency: general_currency_index::<T>(foreign_currency),
 								currency_payout: invest_amount_foreign_denominated / 2,
-								// TODO(@luis): Should be `invest_amount_pool_denominated / 4` with
-								// deltas because we have processed 50% of the investment with
-								// conversion rate 50%
```

Comment:
The conversion rate used here is only for pool <-> tranches. Not for pool <-> foreign. Actually we do not have ITs with prices other than 1:1 for pool <-> foreign currencies. (We should)